oshmem/runtime/oshmem_shmem_params.c

```diff
@@ -13,12 +13,37 @@
 #include "oshmem/constants.h"
 
 
+bool oshmem_shmem_abort_print_stack = false;
+int oshmem_shmem_abort_delay = 0;
 int oshmem_shmem_lock_recursive = 0;
 int oshmem_shmem_api_verbose = 0;
 int oshmem_preconnect_all = 0;
 
 int oshmem_shmem_register_params(void)
 {
+    oshmem_shmem_abort_delay = 0;
+    (void) mca_base_var_register("oshmem",
+                                 "oshmem",
+                                 NULL,
+                                 "abort_delay",
+                                "If nonzero, print out an identifying message when abort is invoked (hostname, PID of the process that called abort operation) and delay for that many seconds before exiting (a negative delay value means to never abort).  This allows attaching of a debugger before quitting the job.",
+                                 MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
+                                 OPAL_INFO_LVL_9,
+                                 MCA_BASE_VAR_SCOPE_READONLY,
+                                 &oshmem_shmem_abort_delay);
+
+    oshmem_shmem_abort_print_stack = false;
+    (void) mca_base_var_register("oshmem",
```

Comment:
Should both of these MCA params be aliases to the MPI counterparts?

And/or should we make both the OSHME and MPI counterparts be aliases to underlying OPAL MCA params?